docs/source/reference/fftpack.rst

```diff
@@ -1,7 +1,7 @@
 .. module:: cupyx.scipy.fftpack
 
-Discrete Fourier transforms
-===========================
+Discrete Fourier transforms (``scipy.fftpack``)
```

Comment:
Might be worth saying "legacy" in these docs somewhere, too, and pointing to `scipy.fft` as the preferred method now